### ISSUE TITLE
Feature: Visual indicator of rewind state

### ISSUE THREAD

**Issue by NitrOP7674** [NitrOP7674/FreeplayCheckpoint#24]
Idea: a bar at the bottom of the screen with a position indicator to show where you are in the history while rewinding/fast-forwarding.

It can be either a simple, wide rectangle with a triangle that slides left/right along it, or it would be crazy to show tiny thumbnails of the state from the times like video editing software does (example below).  I'm not sure if capturing the whole screen in order to resize it is possible, however.

![image](https://user-images.githubusercontent.com/78051141/165002644-0300f362-13c0-4199-9cec-fe366e785e20.png)

We could also mark where checkpoints were loaded/saved or where the last quick checkpoint was taken.

**Comment by 2-X** [comment]
i don't think you would need to take screenshots. since you have the position data you could create a render kind of like the ones in the 3D replay viewer on ballchasing.com (which currently is broken), but from the calculated camera angle based on the user's camera settings and car position
